### ISSUE TITLE
Docker container fails to start if no command is specified in deploy UI

### ISSUE THREAD

**Issue by petrovicboban** [HubSpot/Singularity#1438]
Images may be built with CMD or ENTRYPOINT, so there is no need to add command as argument while executing 'docker run'  
If I dont supply it in deploy UI, executor breaks, obviously because unhandled exception:

```
java.lang.NullPointerException: null
	at java.lang.StringBuilder.<init>(StringBuilder.java:112) ~[na:1.8.0_121]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.getCommand(SingularityExecutorTaskProcessBuilder.java:97) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.buildProcessBuilder(SingularityExecutorTaskProcessBuilder.java:110) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.call(SingularityExecutorTaskProcessBuilder.java:79) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.call(SingularityExecutorTaskProcessBuilder.java:24) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_121]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_121]
INFO  [2017-03-01 12:05:47,450] [rocessBuilder-0] web-registry.lalala.1488369941270.1.7d 934 - Sending status update "NullPointerException while initializing task: null" (TASK_LOST)
```

**Comment by petrovicboban** [comment]
Just to make things more clear:  
if I let this field empty while deploy custom executor running docker container
![image](https://cloud.githubusercontent.com/assets/13204754/23650197/90c4e2f6-0321-11e7-8f47-21999c7275b9.png)
task fails to start, it ends in LOST state.

![image](https://cloud.githubusercontent.com/assets/13204754/23650259/cf6f74e4-0321-11e7-9ec4-f4e5ff96d3d0.png)


with following error in executor.java.log:
```
java.lang.NullPointerException: null
	at java.lang.StringBuilder.<init>(StringBuilder.java:112) ~[na:1.8.0_121]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.getCommand(SingularityExecutorTaskProcessBuilder.java:97) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.buildProcessBuilder(SingularityExecutorTaskProcessBuilder.java:110) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.call(SingularityExecutorTaskProcessBuilder.java:79) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at com.hubspot.singularity.executor.task.SingularityExecutorTaskProcessBuilder.call(SingularityExecutorTaskProcessBuilder.java:24) ~[SingularityExecutor-0.13.0-shaded.jar:0.13.0]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_121]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_121]
```

**Comment by tpetr** [comment]
This was fixed in [0.14.0](https://github.com/HubSpot/Singularity/releases/tag/Singularity-0.14.0) via https://github.com/HubSpot/Singularity/pull/1401.
